lib/web_ui/lib/src/engine/text/layout_service.dart

```diff
@@ -712,13 +618,12 @@ class SpanBox extends RangeBox {
     required ui.TextDirection boxDirection,
     required this.contentDirection,
     required this.isSpaceOnly,
-  })  : span = spanometer.currentSpan,
+  })  : span = spanometer.currentSpan as FlatTextSpan,
```

Comment:
Is there a way to remove the need for a runtime cast here? This feels like an attempt to restore previously known information.

Reply:
This code has been deleted.